NSS-Day-Cohort-66/rare-api-dapper-pirates
# Kq newest branch

Within the `categories.py` module I created a Django method named `def update` that handles the ability to edit/PUT a single category.

### Steps to Test
1) Pull down the `kq-newestBranch` (I understand I could have named this branch better)
2) Ensure debugger is running
3) In Postman make sure there is `Authorization` selected and a valid `Token`.
4) select **PUT**, **Body**, **JSON**
5)In the address bar ensure you have this: http://localhost:8000/categories/1   (Or whatever id you choose)6
6) In the body syntax should appear as such:
 {
        "label": "Dogs"
    }
## Edit the name to what you want
7) Click `Send` and you should receive `Status: 204 No Content` if successful
8) Now, conduct a `GET` of http://localhost:8000/categories and ensure the id was edited

# After this is complete, head to client side and review the PR. This PR supports the server side and feeds into the client side
